src/fparser/tests/test_parser.py

```diff
@@ -195,9 +195,23 @@ def test_deallocate():
 
 def test_moduleprocedure():
     assert_equal(parse(ModuleProcedure,\
+                       'Procedure a'), 'MODULE PROCEDURE a')
+    assert_equal(parse(ModuleProcedure,\
+                       'procedure a , b'), 'MODULE PROCEDURE a, b')
+    assert_equal(parse(ModuleProcedure,\
+                       'procedure :: a '), 'MODULE PROCEDURE a')
+    assert_equal(parse(ModuleProcedure,\
+                       'procedure :: a , b'), 'MODULE PROCEDURE a, b')
+    assert_equal(parse(ModuleProcedure,\
                        'ModuleProcedure a'), 'MODULE PROCEDURE a')
     assert_equal(parse(ModuleProcedure,\
                        'module procedure a , b'), 'MODULE PROCEDURE a, b')
+    assert_equal(parse(ModuleProcedure,\
+                       'module procedure :: a '), 'MODULE PROCEDURE a')
+    assert_equal(parse(ModuleProcedure,\
+                       'module procedure :: a , b'), 'MODULE PROCEDURE a, b')
+    assert_equal(parse(ModuleProcedure,\
+                       'moduleprocedure::a,b'), 'MODULE PROCEDURE a, b')
```

Comment:
Extend existing test to check that fparser is happy with/without the "module" prefix and/or the "::" separator. Great.

Reply:
As per the review guidelines (https://github.com/stfc/fparser/wiki/CodeReview), please could you update this routine to be pep8 and pylint compliant (i.e., two blank lines before and after it, a docstring etc.). Also, we're trying to move away from using nose so please could you replace all uses of "assert_equal" with just "assert" in this routine.